geode-docs/tools_modules/gfsh/command-pages/export.html.md.erb

```diff
@@ -183,7 +183,9 @@ Latest\server1\region2_20121001.gfd on host : 192.0.2.0
 
 Export logs to a given directory.
 
-All files that have logs in the specified time range will be exported. If no time range is specified, all logs will be exported.
+All files that have logs in the specified time range will be exported. If no time range is specified, all logs will be exported. It should be noted that when specifying an `--end-time` argument using a date only, the time is assumed to be `00:00`, meaning that no logs that were written on the date provided will be exported, only those written up until `23:59:59.999` on the day prior.
+
+When using the `yyyy/MM/dd/HH/mm/ss/SSS/z` format for `--start-time` or `--end-time`, the time zone, denoted `z`, should be specified as either a 3-letter time zone such as `PST` or as an offset to GMT/UTC such as `GMT+08:00`.
```

Comment:
I'd like to see these format descriptions/improvements in the description of the parameters.

I think that your wording is adequate, but I came up with this possibility for a rewrite of the added info for the end time info.  Use it if you like.
"An end time specified by only a date implements a time of `00:00`. This exports logs written up until `23:59:59.999` on the date prior to the one specified.

Reply:
I've moved the format descriptions to the table section as requested, and used your rewrite, since it's far more concise and clear.